src/daemon/daemon.cpp

```diff
@@ -1184,8 +1198,9 @@ mp::Daemon::Daemon(std::unique_ptr<const DaemonConfig> the_config)
           mp::utils::backend_directory_path(config->data_directory, config->factory->get_backend_directory_name()),
           mp::utils::backend_directory_path(config->cache_directory, config->factory->get_backend_directory_name()))},
       daemon_rpc{config->server_address, *config->cert_provider, config->client_cert_store.get()},
-      instance_mod_handler{register_instance_mod(vm_instance_specs, operative_instances, deleted_instances,
-                                                 preparing_instances, [this] { persist_instances(); })}
+      instance_mod_handler{register_instance_mod(
+          vm_instance_specs, operative_instances, deleted_instances, preparing_instances,
+          [this] { persist_instances(); }, [] { return get_bridged_interface_name(); })}
```

Comment:
Do you really need the lambda here? Unlike to persist instances, there is no closure for the bridge, so you should be able to pass the function directly.

Reply:
Yep, nice catch. `[this] { persist_instances(); }, get_bridged_interface_name)}` will do it.

Reply:
Right, changed.

Reply:
:+1: